### PR TITLE
get data from DB fixed #2  

### LINKED ISSUE #2: User Journeys : Home Page
As a user I can see all services that related to parties and slider of service photos.

### PR DESCRIPTION
**Pull request by amanshawar**
- [x] get Data From DB and show it

### CODE REVIEW

File: src/backEnd/server.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ db.authenticate()
   .then(()=>console.log('DB connected'))
   .catch(err => console.log("err"));
 
-app.get('/',(req,res)=>{res.send("INDEX");});
+//app.get('/',(req,res)=>{res.send("INDEX");});
```
Comment (shahenazmonia) [inline src/backEnd/server.js]: delete unrequired lines

File: src/backEnd/controllers/services.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const Sequelize = require('sequelize');
+const services = require('../../database/models/services');
+exports.get = (req, res) => {
+  services.findAll({
+  attributes: ['name'],
+   raw: true
+  })
+  .then((result)=>{
+      console.log("done",result);
```
Comment (shahenazmonia) [inline src/backEnd/controllers/services.js]: delete console log

File: src/backEnd/server.js (hunk 1 above)
Comment (amanshawar) [reply src/backEnd/server.js]: done i deleted

File: src/backEnd/controllers/services.js (hunk 2 above)
Comment (amanshawar) [reply src/backEnd/controllers/services.js]: ok

**Pull request merged**